js/main.js

```diff
@@ -402,3 +403,15 @@ function isDone(directions) {
   // If there are no departures and they all have isDone == true, we're done
   return true;
 }
+
+function checkOutdatedSource(){
+  $.ajax({
+    url: 'https://api.github.com/repos/umts/BusInfoBoard',
+    success: function(repoData){
+      lastRepoUpdate = moment(repoData.updated_at);
+      lastSourceUpdate = moment(document.lastModified);
```

Comment:
Interesting that GH pages sets this response header to the most recent commit.  This is, I think, a side-effect of the pages/jekyl build process.